packages/dds/tree/src/feature-libraries/chunked-forest/codec/nodeShape.ts

```diff
@@ -33,11 +33,11 @@ export class NodeShape extends Shape<EncodedChunkShape> implements NodeEncoder {
 	public constructor(
 		public readonly type: undefined | TreeNodeSchemaIdentifier,
 		public readonly value: EncodedValueShape,
-		public readonly fields: readonly KeyedFieldEncoder[],
+		public readonly fieldEncoders: readonly KeyedFieldEncoder[],
```

Comment:
While I think this is an improvement, I think there is a slight risk of confusion here now that someone might think these are just some set of field encoders this node can use, not that they are encoders for each field.

Adding a comment can clarify, and get us the best of both:

```suggestion
		/**
		 * Encoders for a specific set of fields, by key, in the order they will be encoded.
		 * Any fields not included here will be encoded using {@link NodeShape.extraLocal}:
		 * If `extraLocal` is undefined, then these must handle all non-empty fields.
		 */
		public readonly fieldEncoders: readonly KeyedFieldEncoder[],
```